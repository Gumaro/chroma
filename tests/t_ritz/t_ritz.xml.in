<?xml version="1.0"?>
<params>
  <!-- Lattice Dimensions etc -->
  <lattice>
    <nrow>4 4 4 8</nrow>
    <boundary>1 1 1 -1</boundary>
  </lattice>

  <!-- RNG Seed for initing Gaussians and the like -->
  <RNG>
    <seed>11 0 0 0</seed>
  </RNG>

  <!-- Start up config details -->
  <Cfg>
     <!-- This will probably soon be standardised. 2 means from file  -->
     <startType>2</startType>
     <gaugeFilename>../test_purgaug.cfg1</gaugeFilename>

     <!-- Format = 0 means SZIN format -->
     <gaugeFileFormat>0</gaugeFileFormat>
  </Cfg>
  <quarkMass>-1.4</quarkMass>
  <eig>
     <RsdR>1.0e-5</RsdR>
     <RsdA>1.0e-7</RsdA>
     <RsdZero>5.0e-6</RsdZero>
     <ProjApsiP>false</ProjApsiP>    
     <Neig>4</Neig>
     <MaxKS>100</MaxKS>
     <MaxCG>10000</MaxCG>
  </eig>
</params>
