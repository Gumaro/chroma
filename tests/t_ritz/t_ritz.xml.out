<?xml version="1.0"?>


<ritzTest>
  <params>
    <lattice>
      <nrow>4 4 4 8</nrow>
      <boundary>1 1 1 -1</boundary>
    </lattice>
    <RNG>
      <seed>11 0 0 0</seed>
    </RNG>
    <quarkMass>-1.4</quarkMass>
    <Cfg>
      <startType>2</startType>
      <gaugeFileFormat>0</gaugeFileFormat>
      <gaugeFilename>../test_purgaug.cfg1</gaugeFilename>
    </Cfg>
    <eig>
      <RsdR>1e-05</RsdR>
      <MaxCG>10000</MaxCG>
      <Neig>4</Neig>
      <RsdA>1e-07</RsdA>
      <RsdZero>5e-06</RsdZero>
      <ProjApsiP>false</ProjApsiP>
    </eig>
  </params>
  <GaugeInfo><szin>
      <cfg_version>7</cfg_version>
      <date>Sat Jun 16 00:35:57 2001</date>
      <banner>(Sat Jun 16 00:21:07  2001) Source=MOTIE Target=linux Release= Quarks=g</banner>
      <FermTypeP>0</FermTypeP>
      <Nd>4</Nd>
      <Nc>3</Nc>
      <BetaMC>20.1</BetaMC>
      <BetaMD>0</BetaMD>
      <KappaMC>0</KappaMC>
      <KappaMD>0</KappaMD>
      <MassMC>0</MassMC>
      <MassMD>0</MassMD>
      <dt>0</dt>
      <MesTrj>1</MesTrj>
      <TotalCG>0</TotalCG>
      <TotalTrj>2</TotalTrj>
      <spec_acc>0</spec_acc>
      <NOver>2</NOver>
      <TotalTry>12288</TotalTry>
      <TotalFail>56</TotalFail>
      <Nf>9.48851e+33</Nf>
      <Npf>2</Npf>
      <RefMomTrj>302059064</RefMomTrj>
      <RefFnoiseTrj>36</RefFnoiseTrj>
      <LamPl>9.45468e+33</LamPl>
      <LamMi>0</LamMi>
      <AlpLog>4.2039e-45</AlpLog>
      <AlpExp>4.2039e-45</AlpExp>
      <nrow>4 4 4 8</nrow>
      <seed>
        <Seed>
          <elem>811</elem>
          <elem>1073</elem>
          <elem>1535</elem>
          <elem>1310</elem>
        </Seed>
      </seed>
    </szin>

  </GaugeInfo>
  <plaquette>
    <w_plaq>0.902263</w_plaq>
    <s_plaq>0.862579</s_plaq>
    <t_plaq>0.941947</t_plaq>
    <link>0.734641</link>
  </plaquette>
<EigSpecRitzCG>
  <eval1>
    <n_count>882</n_count>
  </eval1>
  <eval2>
    <n_count>836</n_count>
  </eval2>
  <eval3>
    <n_count>651</n_count>
  </eval3>
  <eval4>
    <n_count>1471</n_count>
  </eval4>
  <eValues>
    <lambda>0.447117 0.44946 0.453732 0.458871</lambda>
  </eValues>
  <n_cg_tot>3840</n_cg_tot>
</EigSpecRitzCG>

</ritzTest>